crates/ty_server/src/server/api/semantic_tokens.rs

```diff
@@ -25,7 +24,10 @@ pub(crate) fn generate_semantic_tokens(
     let mut prev_start = 0u32;
 
     for token in &*semantic_token_data {
-        let lsp_range = token.range().to_lsp_range(&source, &line_index, encoding);
+        let lsp_range = token
```

Comment:
We'll need to limit the semantic tokens to a single range but I'll do this in a separate PR